src/components/modal/EditorModal.jsx

```diff
@@ -0,0 +1,180 @@
+    }
+  };
+
+  const clickHandler = () => {
+    if(note.title === ""){
+      AlertToast("Please Enter Note Title")
+      return;
+    }
+    if(note.content === ""){
+      AlertToast("Please Enter note text")
+      return;
+    }
+   
+    setEnable(false);
+    const parsedData = parse(`${note.content}`).props.children;
+    const today = new Date();
+    console.log(today)
+    var timeStamp =
+      today.getFullYear() +
+      "-" +
+      (today.getMonth() + 1) +
+      "-" +
+      today.getDate() +
+      " " +
+      today.getHours() +
+      ":" +
+      today.getMinutes() +
+      ":" +
+      today.getSeconds();
+    dispatch(
+      createNoteHandler({
+        title: note.title,
+        content: parsedData,
+        timestamp: timeStamp,
+        bgcolor: bgColor,
+        priority: note.priority,
+        tags:tags,
+      })
+    );
+    dispatch(handleToggleModal());
+  };
+
+  return (
+    <>
+      <EditorWrapper style={{ backgroundColor: bgColor }}>
+        <CloseButton onClick={() => dispatch(handleToggleModal())}>
+          {" "}
+          X
+        </CloseButton>
+
+        <TitleBox
+          type="text"
+          placeholder="Add title ....."
+          style={{ backgroundColor: bgColor }}
+          value={note.title}
+          name="title"
+          onChange={changeHandler}
+        />
+        <ReactQuill
+          placeholder={"Add notes......"}
+          modules={enable ? modules : noModules}
+          height={"100px"}
+          value={note.content}
+          readOnly={!enable}
+          onChange={(e) => setNote((prev) => ({ ...prev, content: e }))}
+        />
+        <EditorFooter>
+          <Pallette>
+            <PinkButton onClick={() => setbgColor("lightpink")}></PinkButton>
+            <BlueButton onClick={() => setbgColor("lightblue")}></BlueButton>
+            <GreenButton onClick={() => setbgColor("lightgreen")}></GreenButton>
+            <YellowButton onClick={() => setbgColor("yellow")}></YellowButton>
+            <WhiteButton onClick={() => setbgColor("white")}></WhiteButton>
+          </Pallette>
+          <div>
+            <CheckBoxInput
+              type="checkbox"
+              checked={tags.some((tag) => tag === "Office")}
+              value="Office"
+              id="office"
+              onChange={setTagsHandler}
+            />
+            <Checkboxlabel htmlFor="office">Office</Checkboxlabel>
+            <CheckBoxInput
+              type="checkbox"
+              checked={tags.some((tag) => tag === "Health")}
+              value="Health"
+              id="health"
+              onChange={setTagsHandler}
+            />
+            <Checkboxlabel htmlFor="health">Health</Checkboxlabel>
+            <CheckBoxInput
+              type="checkbox"
+              checked={tags.some((tag) => tag === "Study")}
+              value="Study"
+              id="study"
+              onClick={setTagsHandler}
+            />
```

Comment:
It's ok for now. But later on use a single checkbox component and map over some array.

Reply:
okay, will do that later